dbt/include/databricks/macros/materializations/incremental/strategies.sql

```diff
@@ -60,7 +60,7 @@
       {%- if dest_col in source_columns -%}
         {%- do common_columns.append(dest_col) -%}
       {%- else -%}
-        {%- do common_columns.append('NULL') -%}
+        {%- do common_columns.append('DEFAULT') -%}
```

Comment:
Hi,
This seems to be a breaking change, "NULL" (String value) or "DEFAULT" (String value) won't solve the purpose. For all the unmatched columns, we will end up having multiple "DEFAULT" (String) values as columns that can raise conflict/ambiguous column error.

Default value works when we don't pass the column in the argument of insert statement. Could you please try [this simple resolution](https://github.com/databricks/dbt-databricks/pull/604#discussion_r1515123621):

` {%- set common_columns_csv = set(source_columns).intersection(dest_columns) | sort | join(', ') -%}`

Reply:
good call

Reply:
They're not inserted as strings, but as literals.

Example from test run:
```
insert into table `peco`.`test17110372962014884605_test_incremental_on_schema_change`.`incremental_append_new_columns_remove_one` (`id`, `field1`, `field2`, `field3`, `field4`)
select `id`, `field1`, DEFAULT, `field3`, `field4` from `incremental_append_new_columns_remove_one__dbt_tmp`
```